recipes/basic/meta.yaml

```diff
@@ -1,10 +1,10 @@
 package:
   name: basic
-  version: "1.4.0"
+  version: "1.4.1"
 
 source:
-  url: https://github.com/akds/BASIC/archive/v1.4.0.tar.gz
-  sha256: f6afcd5d53f06582effe5f987046ae5bf1e5d6087c8202e3ea13853ff465cd75
+  url: https://github.com/akds/BASIC/archive/v1.4.1.tar.gz
+  sha256: dc898f0b9f6491a69521bef82306d950f16566e4d0d6b3809b11725f707ba133
 
 build:
   number: 1
```

Comment:
@dcroote , since you are updating the package to a new version, the build number should be 0. Please see "Build number and string", https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html, for more details.

Reply:
Good catch @npavlovikj, sorry about that.